### ISSUE TITLE
[BUG] EverythingPowerToys query crashes (hangs) Everything

### ISSUE THREAD

**Issue by raleik-pl** [lin-ycv/EverythingPowerToys#73]
**Describe the bug**
If I run the query using PowerToys Run plugin, I get no results and Everything app crashes (hangs-up).
Then I have to restart Everything service and app to get it work again (Everything, not PowerToys Run plugin)

**To Reproduce**
1. Open PowerToys Run window
2. Type in Everyting query (in my case starting with "?" to activate plugin)
3. No results are shown in PowerToys Run box
4. Open Everything app
5. Observe that it's not responding.

**Expected behavior**
Search query results shown in PowerToys Run box

**Screenshots**

**Logs (please upload or provide link to you log):**
 Logs can be generated by right clicking the taskbar icon > Report Bug
or be found at
```
[2024-01-11 09:49:02.2454] [WARN] [C:\a\_work\1\s\src\modules\launcher\PowerLauncher\Plugin\PluginManager.cs::230]
PluginManager.QueryForPlugin Everything. Query cost - 226247 milliseconds

[2024-01-11 09:49:02.2454] [WARN] [C:\a\_work\1\s\src\modules\launcher\PowerLauncher\Plugin\PluginManager.cs::230]
PluginManager.QueryForPlugin Everything. Query cost - 225766 milliseconds

[2024-01-11 09:49:02.2454] [WARN] [C:\a\_work\1\s\src\modules\launcher\PowerLauncher\Plugin\PluginManager.cs::230]
PluginManager.QueryForPlugin Everything. Query cost - 226439 milliseconds

[2024-01-11 09:49:02.2454] [WARN] [C:\a\_work\1\s\src\modules\launcher\PowerLauncher\Plugin\PluginManager.cs::230]
PluginManager.QueryForPlugin Everything. Query cost - 225326 milliseconds
```
[PowerToysReport_2024-01-11-09-52-54.zip](https://github.com/lin-ycv/EverythingPowerToys/files/13900705/PowerToysReport_2024-01-11-09-52-54.zip)

**Version (please provide the version of software you are using):**
 - PowerToy:  v0.77.0
 - Everything: 1.4.1.1024 (x64)
 - EverythingPowerToys: v0.77.0.0
 - Windows: 11 22H2 22621.3007

**Additional context**

**Comment by lin-ycv** [comment]
I'm still trying to reproduce this, but looking at your log:
```
[2024-01-11 09:32:50.9870] [ERROR] [D:\Repo\PowerToys\src\modules\launcher\Plugins\EverythingPowerToys\Main.cs::181]
-------------------------- Begin exception --------------------------
Message: Everything Exception

Exception full name  : System.ArgumentException
Exception message    : title cannot be null or empty (Parameter 'title')
Exception stack trace:
   at Wox.Plugin.ToolTipData..ctor(String title, String text)
   at Community.PowerToys.Run.Plugin.Everything.Everything.Query(String query, Settings setting)+MoveNext()
   at System.Collections.Generic.List`1.AddRange(IEnumerable`1 collection)
   at Community.PowerToys.Run.Plugin.Everything.Main.Query(Query query, Boolean delayedExecution)
Exception source     : Wox.Plugin
Exception target site: Void .ctor(System.String, System.String)
Exception HResult    : -2147024809
-------------------------- End exception --------------------------
```
it looks like Everything is have problems returning the result of the query, or it's returning something that's not being parsed properly.
Does your query have any special characters or filters?
Does your query work when using Everything and not this plugin?

**Comment by raleik-pl** [comment]
Hi again,
I'm only using plain ascii chararcters in the query.
I'm not sure anymore, but I think it also crashed when searched directly in Everything, which would actually make it Everything's bug, not plugin's.

It seems that it could be caused by my changes to the index I did a bit before. I only left "Index recent changes" and disabled all other options as I don't use thaem anyway.
![image](https://github.com/lin-ycv/EverythingPowerToys/assets/56969220/a5c0b828-0eae-48c8-b6bf-d29d74e59e77)
The issue started after that.

Now I restored the defaults and rebuilt the index and now it all works fine again.
![image](https://github.com/lin-ycv/EverythingPowerToys/assets/56969220/5609d5e4-a99e-46de-a0e9-0279445f4ebd)
